core/distributed/collective_communicator.cpp

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: 2024 - 2025 The Ginkgo authors
```

Comment:
I think the range of years is somehow incorrect ? I think it should 2017 - 2025 ? Or is it different for each file ?

Reply:
IMO it's not important that every file has the copyright 2017-2025. This file didn't exist 2017, so putting a copyright for that year into it seems a bit pointless. Also this is just what the reuse tool does, so we should stick to it.

Reply:
Yes, I think that is fine, but `neighborhood_communicator.cpp` also didnt exist in 2017. So, I would prefer that we be uniform.